Client/core/ServerBrowser/CServerBrowser.cpp

```diff
@@ -2247,6 +2272,29 @@ bool CServerBrowser::OnServerListChangeRow(CGUIKeyEventArgs Args)
     return true;
 }
 
+bool CServerBrowser::OnServerListKeyDown(CGUIKeyEventArgs Args)
+{
+    switch (Args.scancode)
+    {
+        case DIK_UPARROW:
+        case DIK_DOWNARROW:
+        {
+            OnServerListChangeRow(Args);
+            break;
+        }
+        // Remove selected server from recent list when pressing on backspace
+        case DIK_BACK:
```

Comment:
I'd add DIK_DELETE since its very common on Windows. Backspace is more common in macOS but both should be fine here imo